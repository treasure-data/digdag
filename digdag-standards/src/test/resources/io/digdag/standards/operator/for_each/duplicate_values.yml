_command:
  a: [1, 1]
_do:
  echo>: value is ${a}
