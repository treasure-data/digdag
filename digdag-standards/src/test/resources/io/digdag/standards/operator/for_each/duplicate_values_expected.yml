+for-0=a=0=1:
  echo>: value is ${a}
  _export: {a: 1}
+for-0=a=1=1:
  echo>: value is ${a}
  _export: {a: 1}
